labgrid/pytestplugin/fixtures.py

```diff
@@ -113,5 +114,9 @@ def env(request):
 def target(env):
     """Return the default target `main` configured in the supplied
     configuration file."""
-    target = env.get_target()
+    try:
+        target = env.get_target()
+    except UserError as e:
```

Comment:
Why do you limit this to UserErrors? We probably can't recover on other errors as well.

Reply:
Yes, that is true. Do we want to also exit on `target is None`? What's the reasoning behind `get_target()` returning `None` anyway? Shouldn't this be an exception instead? Or is returning `None` the workaround to handle a unsuccessful `get_target()` during test runtime?

Reply:
get_target() returning None can also be treated as fatal here, but with an explicit error message. Using pytest.exit here should be ok for now, as this fixture is used very early.